docs/content/useSafeTimeout.mdx

```diff
@@ -0,0 +1,32 @@
+---
+title: useSafeTimeout
+---
+
+`useSafeTimeout` is a utility Hook that allows you to safely call `setTimeout` and `clearTimeout` within a component, ensuring that all timeouts are cleared when the component unmounts.
```

Comment:
Does `utility hook` sound good here for language??

Reply:
Yeah, I think it's fine. I've only made a conscious effort to be precise with classification of _behaviors_ into "generic behaviors" and "component behaviors." This doesn't really fall into the category of behavior though.